libs/ledgerjs/packages/hw-app-btc/src/splitTransaction.ts

```diff
@@ -100,18 +104,16 @@ export function splitTransaction(
     });
   }
 
-  let witnessScript, locktime;
-
   if (witness) {
     witnessScript = transaction.slice(offset, -4);
     locktime = transaction.slice(transaction.length - 4);
-  } else {
+  } else if (!zcashv5) {
     locktime = transaction.slice(offset, offset + 4);
   }
 
   offset += 4;
 
-  if (overwinter || isDecred) {
+  if ((overwinter || isDecred) && !zcashv5) {
```

Comment:
Not very important but this condition looks weird, overwinter and zcash should probably be in the same condition.
Maybe: `if ((overwinter && !zcashv5) || isDecred)`
But really not important :smile: